pyfolio/round_trips.py

```diff
@@ -260,15 +252,21 @@ def extract_round_trips(transactions,
                           columns=['portfolio_value'])\
             .assign(date=portfolio_value.index)
 
-        roundtrips['date'] = roundtrips.close_dt.apply(lambda x:
-                                                       x.replace(hour=0,
-                                                                 minute=0,
-                                                                 second=0))
-
-        tmp = roundtrips.join(pv, on='date', lsuffix='_')
-
-        roundtrips['returns'] = tmp.pnl / tmp.portfolio_value
-        roundtrips = roundtrips.drop('date', axis='columns')
+        roundtrips['date'] = roundtrips.close_dt.apply(
```

Comment:
Does `roundtrips.close_dt.normalize()` work?

Reply:
I'd imagine not, as close_dt would return a Series.
You could `.normalize()` within the lambda? But I still think you'd run into a tz-aware issue as my reading of normalize docs, it does not change the awareness of the TS.

Assuming the problem here is the dep pytz (and generally that this lambda is pretty lacking style in general)

Reply:
OK, there's probably a reason for this pattern here.

Reply:
which pattern exactly?

The only real add as the tz-aware a reordering and then a removal of: `t = t.reset_index()` which is what caused the collision.

Reply:
Just saying, `.normalize()` and `.tz_localize('utc')` should to the same thing, just vectorized.